packages/functoria-runtime/functoria-runtime.2.1.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "1.2"
+maintainer:   "Gabriel Radanne <drupyog@zoho.com>"
```

Comment:
It seems @drup wants to relinquish maintainance. See https://github.com/ocaml/opam-repository/issues/7868#issuecomment-312624681